addons/account/report/account_aged_partner_balance.py

```diff
@@ -112,7 +113,7 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
                 dates_query += ' BETWEEN %s AND %s)'
                 args_list += (periods[str(i)]['start'], periods[str(i)]['stop'])
             elif periods[str(i)]['start']:
-                dates_query += ' >= %s)'
+                dates_query += ' > %s)'
```

Comment:
Not sure to understand why you are doing this, can you explain? :)

Reply:
We don't need to show current date data in 1-30 column, there is a saperate column 'Not due on' for it. As we are showing data from 1-30 in second column, the dates_query i.e. '(COALESCE(l.date_maturity,l.date)' must be grater than start_date, thats why we remove '=' sign.

Reply:
Actually, this condition does not even seem to be called on Aged Receivable and Payable... I tested, and could not find any way to get it triggered: all the periods have a start date. The else part of the condition is hence always called for the '+120' column, which has no 'stop'. 

Furthermore, I am pretty sure you should not remove the =. The first if checks whether there is a start and a stop for the period, and then uses the SQL BETWEEN operator if it is the case. BETWEEN in inclusive for both the bounds given to it, so I think we should stay inclusive when we don't have any start date.